### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/design-principles-guide#7]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://pages.18f.gov/design-principles-guide/
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `html > body > div > footer > div > p:nth-child(1) > a`
  - `<a href="https://18f.gsa.gov">18F</a>`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.3:1. Recommendation: change text colour to #0476d2.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `html > body > div > div > aside > a`
  - `<a class="skip-link visuallyhidden focusable" href="#main">Skip to Main Content</a>`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `html > body > div > footer > div > p:nth-child(3) > a`
  - `<a href="https://github.com/18F/design-principles-guide/edit/18f-pages/index.md">Edit this page on GitHub</a>`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `html > body > div > footer > div > p:nth-child(2) > a`
  - `<a href="https://github.com/18F/pages/">18F Pages</a>`
### Summary

**Errors:** 4
